app/controllers/bus_stops_controller.rb

```diff
@@ -81,14 +81,33 @@ def update
     @stop.decide_if_completed_by current_user
     if @stop.save
       flash[:notice] = 'Bus stop was updated.'
-      redirect_to bus_stops_path
+      if params[:commit] == 'Save and next' && params[:route_id]
+        route_id = params[:route_id]
+        next_stop = Route.find(route_id)
+                         .next_stop_in_sequence(@stop, params[:direction])
+        if next_stop
+          redirect_to action: 'edit',
+                      id: next_stop.hastus_id,
```

Comment:
Should this be `hastus_id` or a `bus_stop` `id`?

Reply:
The path relies on `hastus_id` for some reason.

Reply:
When you can show a meaningful identifier, show a meaningful identifier, was more or less my thinking.

Reply:
This should've occurred to me before - can we use the path helper here, like in `by_sequence.haml:13`?